mxfusion/components/variables/variable.py

```diff
@@ -264,7 +264,6 @@ def __pow__(self, y):
         from ..functions.operators import power
         return power(self, y)
 
-    @property
-    def T(self):
+    def transpose(self):
```

Comment:
Could you add the description into the docstring?

Reply:
Done